app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -90,13 +90,17 @@ export default function ManageChannelMembers({
     const hasMoreProfiles = useRef(false);
     const [channelMembers, setChannelMembers] = useState<ChannelMembership[]>(EMPTY_MEMBERS);
     const [searchResults, setSearchResults] = useState<UserProfile[]>(EMPTY);
+    const [page, setPage] = useState(0);
```

Comment:
`page` does not affect the rendering, so it can be a ref. Right now we use its value to trigger an effect, but there is no need to have that logic inside an effect.